network/model.go

```diff
@@ -28,12 +29,31 @@ type CreateGameResponse struct {
 	Actions         []model.PlayerAction                 `json:"actions"`
 }
 
+func NewGameResponse(g model.Game) GameResponse {
+	return GameResponse{
+		ID:              g.ID,
+		Players:         g.Players,
+		PlayerColors:    g.PlayerColors,
+		CurrentScores:   g.CurrentScores,
+		LagScores:       g.LagScores,
+		Phase:           g.Phase,
+		BlockingPlayers: g.BlockingPlayers,
+		CurrentDealer:   g.CurrentDealer,
+		Hands:           g.Hands,
+		Crib:            g.Crib,
+		CutCard:         g.CutCard,
+		PeggedCards:     g.PeggedCards,
+		Actions:         g.Actions,
+	}
+}
+
 type CreatePlayerRequest struct {
 	ID   model.PlayerID `json:"id"`
 	Name string         `json:"name"`
 }
 
-type CreatePlayerResponse struct {
-	ID   model.PlayerID `json:"id"`
-	Name string         `json:"name"`
+type PlayerResponse struct {
```

Comment:
🔮 I imagine we're going to have a lot of things that want a player as a response, and each one will be similar, but not the same (i.e. a player's home page vs. a player in the middle of a game). I'm a fan of having many similar structs that are specifically targetting their one usage (i.e. `GetPlayerHomeResponse` and `GetOpponentResponse` structs)